common/predictive-text/worker/models/dummy-model.ts

```diff
@@ -52,11 +52,29 @@ namespace models {
       return this.configuration;
     }
 
-    predict(transform: Transform, context: Context, injectedSuggestions?: Suggestion[]): Suggestion[] {
+    predict(transform: Transform, context: Context, injectedSuggestions?: Suggestion[]): Distribution<Suggestion> {
+      let makeUniformDistribution = function(suggestions: Suggestion[]): Distribution<Suggestion> {
+        let distribution: Distribution<Suggestion> = [];
+        let n = suggestions.length;
+
+        for(let s of suggestions) {
+          distribution.push({sample: s, p: 1});  // For a dummy model, this is sufficient.  The uniformness is all that matters.
+        }
+
+        return distribution;
+      }
```

Comment:
To not break backwards-compatibility, I suggest letting _all_ models that do not implement `predictWithDistribution()` to return a uniform distribution by default.

Reply:
Yep.  That's why I already implemented this within all existing model templates within this PR:

* `DummyModel`
* `WordListModel`
* `TrieModel`  (in this case, as a stop-gap before returning a weight-based probability instead)